src/daft-core/src/array/ops/trigonometry.rs

```diff
@@ -17,6 +17,9 @@ pub enum TrigonometricFunction {
     ArcTan,
     Radians,
     Degrees,
+    Log2,
+    Log10,
```

Comment:
I wouldn't consider Log operations to be a part of of Trigonometry. It would make sense to make a separate module for this.

Reply:
makes sense, will change